ansible/tasks/check_strapi_image_exists-test-prod.yaml

```diff
@@ -0,0 +1,18 @@
+---
+  - name: Fetch imagestream {{ react_infra_name }}:{{ version }}
```

Comment:
I believe this should be the strapi infraname, unless the two are always the same

Reply:
Good catch. Just the step name, doesn't affect execution. But I will change it.